style.css

```diff
@@ -577,6 +690,100 @@ html {
 /* ---------------------------------------------------------------MEDIA__QUERY__SECTION-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- */
 
 @media (min-width: 768px) {
+  .popup__container {
+    /* width: 1300px;
+    height: 1711px; */
```

Comment:
- [ ] Kindly consider removing commented lines of code for it's not a best practice to have them in your code base.